tools/copyblocks/README.md

```diff
@@ -1,9 +1,38 @@
 # Copyblocks
 
-This program can copy Mimir blocks between two GCS buckets. It is GCS specific and uses special calls to copy
-files between buckets directly, without download to a local system first.
+This program can copy Mimir blocks between two Google Cloud Storage (GCS) buckets or two Azure Blob Storage (ABS) buckets.
+It is specific to these two services and performs server-side copies between buckets.
```

Comment:
Should we be super clear that copying across providers (eg. GCS to ABS) is not supported?

Reply:
I reworded the first sentence to try to make it even clearer:
> This program can copy Mimir blocks server-side between two buckets on the same object storage service provider.